src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const invert = {};
```

Comment:
could've been a more meaningful name
```suggestion
  const invertedRobot = {};
```